charts/v1.0/cray-nls/values.yaml

```diff
@@ -104,7 +104,7 @@ cray-service:
       name: "cray-nls"
       image:
         repository: artifactory.algol60.net/csm-docker/stable/cray-nls
-        tag: 0.9.17
+        tag: 0.9.18-debug5
```

Comment:
I assume if we are going to touch the charts here, the version here should at least match the 0.9.21 above.

Reply:
yep https://github.com/Cray-HPE/cray-nls-charts/pull/26/files